.circleci/config.yml

```diff
@@ -213,6 +213,8 @@ jobs:
             go get cloud.google.com/go/pubsub@v1.6.1
             # Temporarily enforce this version. 1.9.0 is incompatible with go < 1.16
             go get github.com/hashicorp/consul/api@v1.8.1
+            # github.com/hashicorp/vault/sdk < v0.2.1 doesn't compile with go1.14
+            go get github.com/hashicorp/vault/sdk@v0.2.0
```

Comment:
Something is off here, maybe with the comment. Did you mean to say `< v0.2.1` ?

Reply:
Indeed 🤦
Fixed by https://github.com/DataDog/dd-trace-go/pull/1039/commits/9fffb9c80d2e0df3776519cb6764645508ff6fbd 👍